packages/voice/src/app/agent/page.tsx

```diff
@@ -202,7 +202,7 @@ const Button: React.FC<{ onClick: () => void; disabled: boolean; children: React
 const AgentPageComponent: React.FC = () => {
   const searchParams = useSearchParams();
   const agentId = searchParams.get('agent') || 'dr-donut';
-  const agentVoice = getAgent(agentId).ttsVoice;
+  const agentVoice = getAgent(agentId)?.ttsVoice;
```

Comment:
I have a couple tweaks incoming to let this work with fixie agent UUIDs...

Reply:
ok, will sit tight and wait for your PR

Reply:
Just pushed it to this branch :P

Reply:
Namely, I also tweaked it to ensure that speculative execution is disabled